guide/source/react-native.md

```diff
@@ -17,22 +17,33 @@ For most projects, since your native app will display the same data and call the
+There are two ways to get started with React Native: using React Native CLI and Expo. 
 
-Here are the downsides to using Expo:
-- You cannot add Native Modules that use Native Code (Java, Swift, etc)
-- You cannot use packages that require linking (these are npm modules that include native code, and allow you to acess native features like the camera, push notifications, fingerprint authentication, etc). \
-- Apps that use Expo are much larger then pure React Native apps
+<h3 id="workflow-native">Use React Native CLI</h3>
 
-Expo does provide some native features ([click here for the full list](https://docs.expo.io/versions/latest/)), but if there is a feature missing that you need, you'll likely need to use an npm package or your own custom native code.
+The React Native guide to get started describes this workflow as following:
 
-You can "eject" your app from Expo to take advantage of Vanilla React Native features, but ejection cannot be undone easily.
+> If you are already familiar with mobile development, you may want to use React Native CLI. It requires Xcode or Android Studio to get started. If you already have one of these tools installed, you should be able to get up and running within a few minutes. If they are not installed, you should expect to spend about an hour installing and configuring them.
 
-The React Native documentation lets you choose between the Expo ("Expo CLI") and Vanilla React Native ("React Native CLI") setup instructions. You can read through the installation instructions and decide which option makes more sense for you.
+Here is the link to the React Native getting started documentation: https://reactnative.dev/docs/environment-setup?guide=native
 
-Here is the link to the React Native getting started documentation: https://reactnative.dev/docs/environment-setup
+<h3 id="workflow-expo">Use Expo</h3>
 
-Once you have your environment setup and have your app running on your device or in the emulator, you can proceed to the next step of the guide: "Meteor React Native Installation"
+The React Native guide to get started describes this workflow as following:
+
+> If you are new to mobile development, the easiest way to get started is with Expo Go.
+
+Expo is an [open-source framework](https://github.com/expo/expo) for apps that run natively on Android, iOS, and the web. The `expo` npm package enables a suite of incredible features for React Native apps. The `expo` package can be installed in nearly any React Native project. See [what Expo offers](https://docs.expo.dev/core-concepts/) for more information or see [why does Expo have its own SDK](https://docs.expo.dev/faq/#why-does-expo-have-its-own-sdk).
```

Comment:
Personally I'd remove the world "incredible" from the second sentence.  Maybe "productive" or "useful" or possibly "powerful"....but "incredible" is not informative and risks showing bias.